### DISCUSSION TITLE
Another discussion platform? Really?

### DISCUSSION THREAD

**Discussion by boraozgen** [ARMmbed/mbed-tools#237] [General]
Do we want even more scattered information? How many forums do we have to check?

I think GitHub Discussions is a direct alternative to Mbed Forums. If they are to be used in parallel, please define the purposes/scopes to avoid confusion.

**Comment by ladislas** [comment]
I think they don't have the same audience and thus the same purpose.

GD is a fairly new addition to GH, people need a GH account to participate, so it's more for people who want to understand how things work under the hood, are ready to make PR's, stuff like that. It's to discuss code, roadmap, features in more details.

Mbed Forum is a general purpose forum, where beginners can ask questions, you can have general discussions, etc. I also think that the seo will be better so people searching on google will find the forum more often than the GD.

What is missing in both place though, is acknowledgement that the core team is reading the discussions on both platform and taking into account the suggestions. The roadmap is still not something very clear for me, so more communication on that would be welcome.

We would also need a way to celebrate and thank hard core contributors such as Zoltan Hudak and Jan Kamidra who are dedicating a lot of time to thoroughly answer all the questions from beginners and experts alike.

**Reply by boraozgen** [reply]
My complaint was that the discussion media are too scattered and hard to keep track of, but these days there is not much activity anyway...

A little off topic but an Mbed meetup would be nice :) I don't know how it could be organized but thats a topic for a forum discussion.

**Reply by 0xc0170** [reply]
> We would also need a way to celebrate and thank hard core contributors such as Zoltan Hudak and Jan Kamidra who are dedicating a lot of time to thoroughly answer all the questions from beginners and experts alike.

I'll do, I've noticed they are very active and helping greatly there!

> A little off topic but an Mbed meetup would be nice :) I don't know how it could be organized but thats a topic for a forum discussion.

There used to be some in US/Japan during some events. I still remember my first one right before Techcon, 8 years back?

**Reply by ladislas** [reply]
Thanks @0xc0170 for the posts ;)

**Reply by JojoS62** [reply]
Thanks also, but I was already sure that the forum is monitored as well.
And ok, GD is closer to the repos and discussions in PRs or Issues can be avoided. Some hot stuff like cmake progress or roadmap topics could also be pinned in Mbed forum.
